app/templates/view-submission.html

```diff
@@ -5,24 +5,23 @@
 
 {% block main -%}
 
-{% if metadata.trad_as %}
-    {% set  trading_as = metadata.trad_as %}
-    {% set  trad_as_open = '<span class="trading_as"> (<span class="trad_as">' %}
-    {% set  trad_as_close = '</span>)</span>' %}
-{% endif %}
-
 <div>
   <h1 class="saturn">{{ _("Submitted answers") }}</h1>
 </div>
 
 <div class="u-mb-s">
     <div class="panel panel--simple panel--success panel--spacious">
-        <p class="mars">{{ _("Your answers were submitted for <span>%(ru_name)s</span>%(trad_as_open)s%(trad_as)s%(trad_as_close)s on %(submitted_time)s",
-            ru_name = metadata.ru_name,
-            trad_as_open = trad_as_open,
-            trad_as = trading_as,
-            trad_as_close = trad_as_close,
-            submitted_time=metadata.submitted_time|format_datetime) }}
+        <p class="mars">
+            {% if metadata.trad_as %}
+                {{ _("Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_time)s",
+                ru_name = metadata.ru_name,
+                trading_as_name = metadata.trad_as,
+                submitted_time=metadata.submitted_time|format_datetime) }}
+            {% else %}
+                {{ _("Your answers were submitted for <span>%(ru_name)s</span> on %(submitted_time)s",
+                ru_name = metadata.ru_name,
+                submitted_time=metadata.submitted_time|format_datetime) }}
+            {% endif %}
```

Comment:
This logic is duplicated, should we make it common?

Reply:
I did think about it, but went with http://wiki.c2.com/?ThreeStrikesAndYouRefactor